acquire/acquire.py

```diff
@@ -946,6 +946,7 @@ class QuarantinedFiles(Module):
         # Sophos
         ("glob", "sysvol/ProgramData/Sophos/Sophos/*/Quarantine"),
         ("glob", "sysvol/ProgramData/Sophos/Sophos */INFECTED"),
+        ("glob", "sysvol/ProgramData/Sophos/Safestore"),
```

Comment:
```suggestion
        ("dir", "sysvol/ProgramData/Sophos/Safestore"),
```
Unless you meant to put a glob pattern in here?